helm/templates/deployer/deployerRole.yaml

```diff
@@ -0,0 +1,213 @@
+apiVersion: authorization.openshift.io/v1
+kind: Role
```

Comment:
tech debt: we have a bunch of resources that can be removed from our rules here in the future (least privilege principle)

Reply:
If we're wrapping the `ServiceAccount` in a toggle, let's make sure to wrap the Role too, eg `{{ if .Values.deployerServiceAccount.enabled }}`

Reply:
I made a new issue ref #83